scanners/nikto/scanner/Dockerfile

```
@@ -2,6 +2,21 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM localhost:5000/sullo/nikto
+FROM alpine:3.14 as build
+ARG scannerVersion
+RUN apk add git
+RUN git clone https://github.com/sullo/nikto.git /nikto \
+  && cd /nikto \
+  && git switch "nikto-$scannerVersion"
```

One liner alternative: 
```suggestion
RUN git clone --depth 1 --branch "nikto-$scannerVersion" https://github.com/sullo/nikto.git
```